libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -179,6 +182,9 @@ - (id) initWithConfig:(SFHybridViewConfig *) viewConfig useUIWebView:(BOOL) useU
     self = [super init];
     if (self) {
         self.useUIWebView = useUIWebView;
+        if (useUIWebView) {
```

Comment:
I'll change it to `WV`. For `%`, we could do query with `WV` over all of `hybrid`. I don't think we need a `not` query, since the remaining `%` will implicitly be using `WK`.